cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -1,6 +1,8 @@
 # Author: Jamie Strandboge <jamie@canonical.com>
 #include <tunables/global>
 
+@{SNAP_MOUNT_DIR_LIST}="{,var/lib/snapd}/snap"
```

Comment:
I think this is missing a leading `/` on the `var/lib/snapd` element - should it instead be:

```
@{SNAP_MOUNT_DIR_LIST}="{,/var/lib/snapd}/snap"
```

Reply:
Indeed, thanks for spotting this!

Reply:
Fixed